pkg/edgecache/manager.go

```diff
@@ -99,7 +106,7 @@ func sendCreateVolume(client cache_volume_service.CacheVolumeClient, account str
 		Volume: &blobVolume,
 	}
 
-	klog.V(2).Infof("calling edge cache CreateBlob")
+	klog.V(2).Infof("calling edge cache CreateBlob %s %s %d", account, container, len(key))
```

Comment:
very minor log message consistency nudge:  edgecache or edge cache or EdgeCache or Edge Cache.  pick one and use it everywhere.

Reply:
updated